site/en/tutorials/audio/simple_audio.ipynb

```diff
@@ -826,7 +826,7 @@
         "x = x[tf.newaxis,...]\n",
         "\n",
         "prediction = model(x)\n",
-        "x_labels = ['no', 'yes', 'down', 'go', 'left', 'up', 'right', 'stop']\n",
+        "x_labels = ['down', 'go', 'left', 'no', 'right', 'stop', 'up', 'yes']\n",
         "plt.bar(x_labels, tf.nn.softmax(prediction[0]))\n",
```

Comment:
```suggestion
        "plt.bar(label_names, tf.nn.softmax(prediction[0]))\n",
```

I think instead of hard-coding them here, we can just use the `label_names` variable that is created earlier, and used throughout this guide (so you need to delete the previous line too).

Reply:
@markmcd @FrankXu1 PTAL